package.json

```diff
@@ -33,7 +33,7 @@
     "mongojs": "2.4.0",
     "redis": "~0.10.1",
     "redis-sharelatex": "^1.0.8",
-    "request": "~2.33.0",
+    "request": "~2.88.2",
     "requestretry": "^1.12.0",
```

Comment:
We should consider upgrading `requestretry` too, see https://github.com/overleaf/document-updater/pull/121#discussion_r380380669 for details.

Reply:
Good catch, updated :+1: